src/main/java/uk/gov/justice/digital/zone/StructuredZoneLoad.java

```diff
@@ -0,0 +1,67 @@
+package uk.gov.justice.digital.zone;
+
+import lombok.val;
+import org.apache.spark.sql.Dataset;
+import org.apache.spark.sql.Row;
+import org.apache.spark.sql.SparkSession;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import uk.gov.justice.digital.config.JobArguments;
+import uk.gov.justice.digital.domain.model.SourceReference;
+import uk.gov.justice.digital.exception.DataStorageException;
+import uk.gov.justice.digital.service.DataStorageService;
+import uk.gov.justice.digital.service.SourceReferenceService;
+
+import javax.inject.Inject;
+
+import static org.apache.spark.sql.functions.col;
+import static uk.gov.justice.digital.converter.dms.DMS_3_4_6.Operation.Load;
+import static uk.gov.justice.digital.converter.dms.DMS_3_4_6.ParsedDataFields.OPERATION;
+
+public class StructuredZoneLoad extends StructuredZone implements DeltaWriter {
+
+    private static final Logger logger = LoggerFactory.getLogger(StructuredZone.class);
+
+    @Inject
+    public StructuredZoneLoad(
+            JobArguments jobArguments,
+            DataStorageService storage,
+            SourceReferenceService sourceReference
+    ) {
+        super(jobArguments, storage, sourceReference);
+    }
+
+    @Override
+    public Dataset<Row> process(SparkSession spark, Dataset<Row> dataFrame, Row table) throws DataStorageException {
+        val filteredRecords = dataFrame.filter(col(OPERATION).equalTo(Load.getName()));
+        return super.process(spark, filteredRecords, table);
+    }
+
+    @Override
+    public void writeValidRecords(
```

Comment:
As previous comments. If these don't need to be public we should make them protected.